data/quickstarts/pachyderm-beginner-tutorial-notebook.yaml

```diff
@@ -0,0 +1,51 @@
+kind: ConsoleQuickStart
+metadata:
+  name: pachyderm-beginner-tutorial-notebook
+  annotations:
+    opendatahub.io/categories: 'AI/Machine learning,Getting started,Jupyter notebook,Pipeline,Python'
+
+spec:
+  displayName: Pachyderm Beginner Tutorial Notebook
```

Comment:
The title might have users thinking this is a tutorial and not a quick start. Maybe we can try this title: "Importing a Pachyderm tutorial notebook"

Reply:
@nadegepepin Could you please review?